certora/helpers/DummyERC20B.sol

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+import "./DummyERC20Impl.sol";
+
+contract DummyERC20B is DummyERC20Impl {}
```

Comment:
Is that needed? is `DummyERC20A` not enough?

Reply:
Sometimes we need to have 2 separate token contracts as they have their own state. I will check if its used and can remove it if its unnecessary.